### PR TITLE
[zh]sync /releases/supported-releases/index.md

### PR DESCRIPTION
**Pull request by yulng**
Sync 1 file with en:
[zh]sync /releases/supported-releases/index.md

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/ok-to-test

File: content/zh/docs/releases/supported-releases/index.md [hunk 1]
```diff
@@ -52,20 +52,7 @@ Istio 控制面可以比数据面高一个版本。但数据面的版本不能
...
-| 1.13       | 否           | 2022 年 2 月 11 日  | 2022 年 10 月 12 日    | 1.20, 1.21, 1.22, 1.23       | 1.16, 1.17, 1.18, 1.19             |
-| 1.12       | 是           | 2021 年 11 月 18 日 | 2022 年 7 月 12 日     | 1.19, 1.20, 1.21, 1.22       | 1.16, 1.17, 1.18                   |
-| 1.11       | 否           | 2021 年 8 月 12 日  | 2022 年 3 月 25 日     | 1.18, 1.19, 1.20, 1.21, 1.22 | 1.16, 1.17                         |
-| 1.10       | 否           | 2021 年 5 月 18 日  | 2022 年 1 月 7 日      | 1.18, 1.19, 1.20, 1.21       | 1.16, 1.17, 1.22                   |
-| 1.9        | 否           | 2021 年 2 月 9 日   | 2021 年 10 月 8 日     | 1.17, 1.18, 1.19, 1.20       | 1.15, 1.16                         |
-| 1.8        | 否           | 2020 年 11 月 10 日 | 2021 年 5 月 12 日     | 1.16, 1.17, 1.18, 1.19       | 1.15                               |
-| 1.7        | 否           | 2020 年 8 月 21 日  | 2021 年 2 月 25 日     | 1.16, 1.17, 1.18             | 1.15                               |
-| 1.6 及更早 | 否           | -                   | -                      | -                            | -                                  |
+{{< support_status_table >}}
```
Comment (windsonsea) [inline content/zh/docs/releases/supported-releases/index.md:55]: Please sync with en version. Don't only remove them in zh version.

File: .golangci.yaml [hunk 2]
```diff
@@ -0,0 +1,105 @@
+# options for analysis running
```
Comment (zirain) [inline .golangci.yaml:1]: why need this?

**Comment by windsonsea** [comment]
1. Need to sign cla
2. Clear lint errors

**Comment by yulng** [comment]
To pass the CLA, I need to register for a Google account. When registering, I was blocked here

企业微信截图_0fb88e6e-7b1e-42a8-9c17-028bd1fcb1bb

**Comment by yulng** [comment]
> To pass the CLA, I need to register for a Google account. When registering, I was blocked here
image

**Comment by ericvn** [comment]
Note that the Google CLA will not be required real soon, and the pipeline will switch to just looking at the EasyCLA. I'm not sure when that will actually happen.

**Comment by yulng** [comment]
> Note that the Google CLA will not be required real soon, and the pipeline will switch to just looking at the EasyCLA. I'm not sure when that will actually happen.

Now it shows
image

**Review by windsonsea — APPROVED** [review]
Thanks.

**Pull request merged**